src/Ledger/Utxo.lagda

```diff
@@ -60,8 +70,6 @@ utxoEntrySizeWithoutVal = 8
 utxoEntrySize : TxOut → MemoryEstimate
 utxoEntrySize o = utxoEntrySizeWithoutVal + size (getValue o)
 
-refScripts : Tx → UTxO → ℙ Script
```

Comment:
why is this going away?

Reply:
Andre decided it would fit better in the Transaction module.  (See [lines 176--179](https://github.com/IntersectMBO/formal-ledger-specifications/pull/376/files/4157be60e9e2c4c5d6e8d24a71dcca5dabd055f0#diff-5c378f0dd9b999246b7c7b3cb945ba44a44760131d755335b715fbc21ff6dedfR176-R179) of Transaction.lagda)